backend/api.py

```
@@ -1907,6 +1907,1047 @@ def process_reaper():
+        if 'notes' in data:
+            updates.append("notes = ?")
+            values.append(data['notes'])
+        
+        updates.append("updated_at = CURRENT_TIMESTAMP")
+        values.append(workspace_id)
+        
+        cursor.execute(f"""
+            UPDATE workspaces 
+            SET {', '.join(updates)}
+            WHERE id = ?
+        """, values)
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)